schema.ts

```diff
@@ -329,9 +324,6 @@ export interface BaseNumberSchema<T extends NumberHintTypes = NumberHintTypes> e
   type: ValueType.Number;
   /** An optional type hint instructing Coda about how to interpret or render this value. */
   codaType?: T;
-
```

Comment:
Note: here and in upload validation I removed some `autocomplete` declarations for numbers and booleans since for the moment we're only supporting strings and objects